src/script/page/LeftSidebar/panels/StartUI/StartUI.tsx

```diff
@@ -100,27 +100,27 @@ const StartUI: React.FC<StartUIProps> = ({
 
   const peopleSearchResults = useRef<SearchResultsData | undefined>(undefined);
 
-  const openFirstConversation = (): void => {
+  const openFirstConversation = async (): Promise<void> => {
     if (peopleSearchResults.current) {
       const {contacts, groups} = peopleSearchResults.current;
       if (contacts.length > 0) {
-        openContact(contacts[0]);
+        await openContact(contacts[0]);
         return;
```

Comment:
you can just return an async function here -> `return openContact(contacts[0]);`